app-modules/basic-needs/database/factories/BasicNeedProgramFactory.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\BasicNeeds\Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use AdvisingApp\BasicNeeds\Models\BasicNeedCategory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\AdvisingApp\Prospect\Models\Program>
+ */
+class BasicNeedProgramFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function getAvailabilityValue()
+    {
+        return collect([
+            $this->faker->numberBetween(1, 24) . ' Hours',
+            $this->faker->numberBetween(1, 100) . ' Days',
+        ])->random();
+    }
+
+    public function definition(): array
+    {
+        return [
+            'name' => $this->faker->name(),
+            'description' => $this->faker->paragraph(),
+            'basic_need_category_id' => BasicNeedCategory::factory(),
+            'contact_person' => $this->faker->name(),
+            'contact_email' => $this->faker->email(),
+            'contact_phone' => $this->faker->numerify('+1 ### ### ####'),
+            'location' => $this->faker->city(),
+            'availability' => $this->getAvailabilityValue(),
```

Comment:
Rather than having a whole separate method for this, you can use `fake()->randomElement`. We use that throughout the application so you should be able to look up an example.